components/Login/Login.test.tsx

```
@@ -0,0 +1,40 @@
+import { fireEvent, render, screen } from '@/test-utils';
+import Login from './Login';
+
+jest.mock('@/components/StravaLoginButton/StravaLoginButton', () => ({
+  StravaLoginButton: ({ onAuthCode }: { onAuthCode: (code: string) => void }) => (
+    <button data-testid="strava-login-btn" onClick={() => onAuthCode('test-code')}>
+      Connect with Strava
+    </button>
```

ESLint

[error] Missing an explicit type attribute for button

(react/button-has-type)